overlord/configstate/configcore/ctrlaltdel.go

```diff
@@ -0,0 +1,141 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
```

Comment:
nitpick: 2021, or if I'll be getting even better at bikeshedding we can make it 2022 already ;-)

Reply:
Fixed.